vNext/shared/AppInsightsCore/src/JavaScriptSDK.Enums/LoggingEnums.ts

```diff
@@ -35,60 +35,61 @@ export const _InternalMessageId = {
     FailedMonitorAjaxOpen: 15,
     FailedMonitorAjaxRSC: 16,
     FailedMonitorAjaxSend: 17,
-    FailedMonitorAjaxGetCorrelationHeader: 18,
-    FailedToAddHandlerForOnBeforeUnload: 19,
-    FailedToSendQueuedTelemetry: 20,
-    FailedToReportDataLoss: 21,
-    FlushFailed: 22,
-    MessageLimitPerPVExceeded: 23,
-    MissingRequiredFieldSpecification: 24,
-    NavigationTimingNotSupported: 25,
-    OnError: 26,
-    SessionRenewalDateIsZero: 27,
-    SenderNotInitialized: 28,
-    StartTrackEventFailed: 29,
-    StopTrackEventFailed: 30,
-    StartTrackFailed: 31,
-    StopTrackFailed: 32,
-    TelemetrySampledAndNotSent: 33,
-    TrackEventFailed: 34,
-    TrackExceptionFailed: 35,
-    TrackMetricFailed: 36,
-    TrackPVFailed: 37,
-    TrackPVFailedCalc: 38,
-    TrackTraceFailed: 39,
-    TransmissionFailed: 40,
-    FailedToSetStorageBuffer: 41,
-    FailedToRestoreStorageBuffer: 42,
-    InvalidBackendResponse: 43,
-    FailedToFixDepricatedValues: 44,
-    InvalidDurationValue: 45,
-    TelemetryEnvelopeInvalid: 46,
-    CreateEnvelopeError: 47,
+    FailedMonitorAjaxSetRequestHeader: 18,
```

Comment:
You can revert this and just add the new id's to the bottom. We use these id's to find common errors internally and do not want to have to account for a shift in them

Reply:
sure, that makes sense